DOL.WHD.Section14c.Web/.eslintrc.js

```diff
@@ -0,0 +1,220 @@
+        "no-undefined": 0,
+        "no-unused-vars": 0,
+        "no-use-before-define": 0,
+
+        // Node.js and CommonJS
+        "callback-return": 0,
+        "global-require": 0,
+        "handle-callback-err": 0,
+        "no-mixed-requires": 0,
+        "no-new-require": 0,
+        "no-path-concat": 0,
+        "no-process-exit": 0,
+        "no-restricted-modules": 0,
+        "no-sync": 0,
+
+        //Stylistic Issues
+        "array-bracket-spacing": 0,
+        "block-spacing": 0,
+        "brace-style": 0,
+        "camelcase": 0,
+        "comma-spacing": 0,
+        "comma-style": 0,
+        "computed-property-spacing": 0,
+        "consistent-this": 0,
+        "eol-last": 0,
+        "func-names": 0,
+        "func-style": 0,
+        "id-length": 0,
+        "id-match": 0,
+        "indent": 0,
+        "jsx-quotes": 0,
+        "key-spacing": 0,
+        "linebreak-style": 0,
+        "lines-around-comment": 0,
+        "max-depth": 0,
+        "max-len": 0,
+        "max-nested-callbacks": 0,
+        "max-params": 0,
+        "max-statements": [2, 30],
+        "new-cap": 0,
+        "new-parens": 0,
+        "newline-after-var": 0,
+        "no-array-constructor": 0,
+        "no-bitwise": 0,
+        "no-continue": 0,
+        "no-inline-comments": 0,
+        "no-lonely-if": 0,
+        "no-mixed-spaces-and-tabs": 0,
+        "no-multiple-empty-lines": 0,
+        "no-negated-condition": 0,
+        "no-nested-ternary": 0,
+        "no-new-object": 0,
+        "no-plusplus": 0,
+        "no-restricted-syntax": 0,
+        "no-spaced-func": 0,
+        "no-ternary": 0,
+        "no-trailing-spaces": 0,
+        "no-underscore-dangle": 0,
+        "no-unneeded-ternary": 0,
+        "object-curly-spacing": 0,
+        "one-var": 0,
+        "operator-assignment": 0,
+        "operator-linebreak": 0,
+        "padded-blocks": 0,
+        "quote-props": 0,
+        "quotes": 0,
+        "require-jsdoc": 0,
+        "semi-spacing": 0,
+        "semi": 0,
+        "sort-vars": 0,
+        "space-after-keywords": 0,
+        "space-before-blocks": 0,
+        "space-before-function-paren": 0,
+        "space-before-keywords": 0,
+        "space-in-parens": 0,
+        "space-infix-ops": 0,
+        "space-return-throw-case": 0,
+        "space-unary-ops": 0,
+        "spaced-comment": 0,
+        "wrap-regex": 0,
+
+        // ECMAScript 6
+        "arrow-body-style": 0,
+        "arrow-parens": 0,
+        "arrow-spacing": 0,
+        "constructor-super": 0,
+        "generator-star-spacing": 0,
+        "no-arrow-condition": 0,
+        "no-class-assign": 0,
+        "no-const-assign": 0,
+        "no-dupe-class-members": 0,
+        "no-this-before-super": 0,
+        "no-var": 0,
+        "object-shorthand": 0,
+        "prefer-arrow-callback": 0,
+        "prefer-const": 0,
+        "prefer-reflect": 0,
+        "prefer-spread": 0,
+        "prefer-template": 0,
+        "require-yield": 0
+    }
+};
```

Comment:
We will look into our current rules and documenting where we derived them from and the reasons for modifications.